src/components/Content/Breadcrumb.css

```diff
@@ -1,19 +1,21 @@
 @import url('../Helper/Helper.css');
 
 .breadcrumbs {
+  overflow: scroll;
   margin-bottom: 20px;
+  white-space: nowrap;
 }
 
 .separator::after {
   color: #e8e8e8;
-  font-size: 22px;
+  font-size: 16px;
   content: ' > ';
   user-select: none;
 }
 
 .level {
   color: #9c9c9c;
-  font-size: 20px;
+  font-size: 15px;
   user-select: none;
 }
 
```

Comment:
You can also remove `breadcrumb:last-child .level {
   color: black;
 }` since this makes no sense anymore.

Reply:
done